### PR TITLE
fix(Geosuggest): Hide the suggest list when the result set is empty

### PR DESCRIPTION
**Pull request by Caerostris**
This fixes a small design bug which was visible when removing focus from the search box while the result set was empty.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,17 +81,6 @@ Default: `''`
...
-Type: `Object`
-Default: `{
-  'input': {},
-  'suggests': {},
-  'suggestItem': {}
-}`
-
-Add an additional style to `Geosuggest`.
-This would support overriding/adding styles to the input suggestList and suggestItem.
-
```
Comment (ro-ka) [inline README.md]: This shouldn’t be removed.

File: README.md [hunk 2]
```diff
@@ -150,7 +139,7 @@ Gets triggered when the input field receives focus.
 
 #### onBlur
 Type: `Function`
-Default: `function(value) {}`
+Default: `function() {}`
```
Comment (ro-ka) [inline README.md]: Same here – do not revert changes made.

**Comment by ro-ka** [comment]
🏁

File: README.md (hunk 2 above)
Comment (Caerostris) [inline README.md]: Yup.

**Pull request merged**